### ISSUE TITLE
Missing double-quote in installer script

### ISSUE THREAD

**Issue by BinToss** [da2x/EdgeDeflector#130]
At Line 67, Column 82
https://github.com/da2x/EdgeDeflector/blob/c725a1e03ab1e48496a8dff24a36c64d6a3f5d5d/EdgeDeflector/resources/nsis_installer.nsi#L67-L68

It should instead read
```nsi
   WriteRegStr HKCU "SOFTWARE\Classes\${PRODUCT}\Capabilities" "ApplicationIcon" "${INSTDIR}\${PRODUCT}.exe,0" 
```

Perhaps the current format works as intended, but consistency and machine-readability is typically preferred.

**Comment by da2x** [comment]
Thanks for the heads up! Fixed in commit 822ab94ce4ff7ed02a6038d8fb5b2583554c4d15. It works without the quote, but it’s a mistake nonetheless.

**Comment by da2x** [comment]
Fixed in [version 1.2.3.0](https://github.com/da2x/EdgeDeflector/releases/tag/v1.2.3.0).
